build.sbt

```diff
@@ -834,6 +834,7 @@ lazy val httpUtils = (project in engine("httpUtils")).
         "org.scala-lang.modules" %% "scala-parser-combinators" % scalaParsersV, // scalaxb deps
         "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingV,
         "com.softwaremill.sttp.client" %% "core" % sttpV,
+        "com.softwaremill.sttp.client" %% "akka-http-backend" % sttpV,
```

Comment:
I don't think we want to introduce akka dependency here, as this module is often used in model.
Let's add it only where it's needed (security or ui)

Reply:
Moved to UI.